.stylelintrc.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
     rules: {
         "block-no-empty": true,
         "color-hex-case": "lower",
-        "max-nesting-depth": 3,
+        "max-nesting-depth": 4,
```

Comment:
Je vois que tu as enlevé les exceptions dans les fichiers que tu as touché, est-ce qu'il a été statué qu'on forçait à 4 maintenant ?